### PR TITLE
Added license and various other enhancements

### PR DESCRIPTION
**Pull request by sabre1041**
* Added LICENSE
* Updated README with details about the chart repository
* Added description to rekor chart
* Upgrade to Rekor server `v0.3.0`

### CODE REVIEW

File: charts/rekor/Chart.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
 name: rekor
-description: A Helm chart for Kubernetes
+description: Rekkor is an API based server for validation and a transparency log for storage
 
 type: application
 
-version: 0.1.2
+version: 0.1.3
```
Comment (cpanato) [inline charts/rekor/Chart.yaml:7]: i would  bump that to 0.2.0 the rekor release is not a bug fix

Comment (cpanato) [reply charts/rekor/Chart.yaml:7]: and a small nit: missing the appVersion in this file

Comment (sabre1041) [reply charts/rekor/Chart.yaml:7]: I added an `appVersion` to the `Chart.yaml` as it separates the lifecycle between the chart and rekor itself

File: charts/rekor/Chart.yaml [hunk 2]
```diff
@@ -1,10 +1,12 @@
 apiVersion: v2
 name: rekor
-description: A Helm chart for Kubernetes
+description: Rekkor is an API based server for validation and a transparency log for storage
```
Comment (cpanato) [inline charts/rekor/Chart.yaml:3]: should be `Rekor`

Comment (sabre1041) [reply charts/rekor/Chart.yaml]: corrected

**Review by cpanato — APPROVED** [review]

**Comment by cpanato** [comment]
thanks for the update 🎉

**Pull request merged**